include/boost/geometry/strategies/cartesian/distance_projected_point_ax.hpp

```diff
@@ -20,6 +20,8 @@
 #ifndef BOOST_GEOMETRY_STRATEGIES_CARTESIAN_DISTANCE_PROJECTED_POINT_AX_HPP
 #define BOOST_GEOMETRY_STRATEGIES_CARTESIAN_DISTANCE_PROJECTED_POINT_AX_HPP
 
+#include <boost/config/pragma_message.hpp>
+BOOST_PRAGMA_MESSAGE("This include file is deprecated and will be removed in Boost 1.87")
```

Comment:
Since this commit will be part of 1.87, should we change the message to 1.88?

Reply:
✅